website/docs/ref/macro.mdx

```diff
@@ -791,6 +791,55 @@ function MyComponent() {
 The `useLingui` React macro is available from **Lingui v5**.
 :::
 
+## Important Notes
+
+### Using Macros
+
+All Core Macros cannot be used at the module level:
+
+```jsx
+import { t } from "@lingui/core/macro";
+
+// ❌ Bad! This won't work because the `t` macro is used at the module level.
+// The `t` macro returns a string, and once this string is assigned, it won't react to locale changes.
+const colors = [t`Red`, t`Orange`, t`Yellow`, t`Green`];
+
+// ✅ Good! Every time the function is executed, the `t` macro will be re-executed as well,
+// and the correctly translated color labels will be returned.
+function getColors() {
+  return [t`Red`, t`Orange`, t`Yellow`, t`Green`];
+}
+```
+
+:::tip
+There is an [ESLint Plugin](/docs/ref/eslint-plugin.md) rule designed to check for this misuse: `t-call-in-function`.
+:::
```

Comment:
Now we can put a direct link to the documentation https://github.com/lingui/eslint-plugin/blob/main/docs/rules/t-call-in-function.md